extensions/ql-vscode/src/common/commands.ts

```diff
@@ -13,6 +13,7 @@ import type {
 import type { QLDebugConfiguration } from "../debugger/debug-configuration";
 import type { QueryTreeViewItem } from "../queries-panel/query-tree-view-item";
 import type { LanguageSelectionTreeViewItem } from "../language-selection-panel/language-selection-data-provider";
+import { Method, Usage } from "../model-editor/method";
```

Comment:
Can we make these type imports?

```suggestion
import type { Method, Usage } from "../model-editor/method";
```